lib/astc_encode.cpp

```diff
@@ -35,19 +35,19 @@
 #include "astc-encoder/Source/astcenc.h"
 
 #if !defined(_WIN32) || defined(WIN32_HAS_PTHREADS)
-#include <pthread.h>
+    #include <pthread.h>
 #else
 // Provide pthreads support on windows
-#define WIN32_LEAN_AND_MEAN
-#include <Windows.h>
+    #define WIN32_LEAN_AND_MEAN
+    #include <Windows.h>
 
 typedef HANDLE pthread_t;
```

Comment:
Can you get it to indent this code up to the #endif?

Reply:
You can exclude lines of code from being formatted by surrounding them with:

```cpp
// clang-format off
...
// clang-format on
```

There are examples of this in the generated files.